canvas_modules/common-canvas/src/notification-panel/notification-panel.scss

```diff
@@ -212,14 +212,14 @@ $notification-panel-button-container-height: 49px;
 	display: inline-flex;
 	color: $text-secondary;
 	svg {
-		width: 10px;
-		height: 10px;
-		margin-right: 3px;
-		margin-top: 3px;
+		width: $spacing-04;
+		height: $spacing-04;
+		margin-right: $spacing-02;
+		margin-top: $spacing-02;
```

Comment:
These are specific values set to make the timestamp a specific size so we dont need to use tokens for these, please revert